Formula/t/tin.rb

```diff
@@ -1,7 +1,7 @@
 class Tin < Formula
   desc "Threaded, NNTP-, and spool-based UseNet newsreader"
   homepage "http://www.tin.org"
-  url "https://www.nic.funet.fi/pub/unix/news/tin/v2.6/tin-2.6.3.tar.xz"
+  url "ftp://ftp.tin.org/pub/news/clients/tin/stable/tin-2.6.3.tar.xz"
```

Comment:
```suggestion
  url "https://sunsite.icm.edu.pl/pub/unix/news/tin/v2.6/tin-2.6.3.tar.xz"
```

If we want to avoid using an FTP, it looks like the mirror in Poland (from the upstream [mirrors page](http://www.tin.org/mirrors.html)) works over HTTP. We were using a mirror from Finland, so it may not be any worse (it seems to load quickly enough for me).